kafka-connect/src/main/java/io/tabular/iceberg/connect/channel/KafkaUtils.java

```diff
@@ -40,5 +46,18 @@ public static ConsumerGroupDescription consumerGroupDescription(
     }
   }
 
+  private static final String WorkerSinkTaskContextClassName =
+          WorkerSinkTaskContext.class.getName();
+
+  @SuppressWarnings("unchecked")
```

Comment:
worth a comment around using reflection to get at some very specific implementation detail stuff here but otherwise :+1: